src/main/io/osd.c

```diff
@@ -633,6 +633,12 @@ static const char * navigationStateMessage(void)
         case MW_NAV_STATE_LAND_START:
             return OSD_MESSAGE_STR("STARTING EMERGENCY LANDING");
         case MW_NAV_STATE_LAND_IN_PROGRESS:
+            if (!navConfig()->general.flags.rth_allow_landing) {
```

Comment:
I don't like this hacky way of figuring out what nav system is doing. We'll need to look into it in 1.9